Extension/package.json

```diff
@@ -1157,6 +1157,12 @@
           "default": "Enabled",
           "description": "%c_cpp.configuration.codeFolding.description%",
           "scope": "window"
+        },
+        "C_Cpp.autocompleteAddParenthesesAfterFunctionCalls": {
```

Comment:
This is a REALLY long setting name.  Can we remove "autocomplete" from the name?  I think it's implied and will at least be described in the description.

Reply:
The longer name matches what other languages use and the UI separates the long text into words, so I'd prefer to keep "autocomplete", but we can see what Julia/others think:
![image](https://user-images.githubusercontent.com/19859882/110180147-1d136280-7dbe-11eb-9847-944412b9dee4.png)

Reply:
Python just calls it "addBrackets" (not saying it's better, but it is shorter)
![image](https://user-images.githubusercontent.com/12818240/110375159-6e159780-8006-11eb-9bba-6d445580019f.png)

I think I would be happy with keeping `autocomplete` in the name if we made it shorter like `autocompleteAddParentheses`.  The description is the better place to add additional detail.  We had similar problems with naming the vcFormatting settings and we just have to rely on the description.

Reply:
Yeah, I like "autoCompleteAddParentheses".